docs/reference-guide/modules/events/pages/event-processors/index.adoc

```diff
@@ -309,21 +293,22 @@ public interface ErrorHandler {
 }
 ----
 
-Based on the provided `ErrorContext` object, you can decide to ignore the error, schedule retries, perform dead-letter-queue delivery, or rethrow the exception.
-
-[[dead-letter-queue]]
-=== Dead-letter queue
-
-Although configuring a <<listener-invocation-error-handler,Listener Invocation Error Handler>>
-and <<event-processor-error-handler,Error Handler>>
-helps you to deal with exceptions when processing events, you still end up in an event handling stop.
-When you only log the error and allow processing to proceed, you will most likely end up with missing data until you fix the predicament and xref:event-processors/streaming.adoc#replaying-events[replay]
-past events.
-If you instead propagate the exception so the event processor keeps retrying, the event processor will stall entirely when the cause is consistent.
-
-Although this behavior is sufficient on many occasions, sometimes it is beneficial if we can unblock event handling by parking the problematic event.
-This is where the Dead-letter Queue comes in. It is a mechanism to park events until they can be processed successfully.
-You can find more information in the xref:axon-framework-reference:events:event-processors/dead-letter-queue.adoc[] section.
+Based on the provided `ErrorContext` object, you can decide to ignore the error, schedule retries, or rethrow the exception.
+
+//[[dead-letter-queue]]
+//=== Dead-letter queue
+//
+// TODO: Reinstate when DLQ API is available in Axon 5.0
+//Although configuring a <<listener-invocation-error-handler,Listener Invocation Error Handler>>
+//and <<event-processor-error-handler,Error Handler>>
+//helps you to deal with exceptions when processing events, you still end up in an event handling stop.
+//When you only log the error and allow processing to proceed, you will most likely end up with missing data until you fix the predicament and xref:event-processors/streaming.adoc#replaying-events[replay]
+//past events.
+//If you instead propagate the exception so the event processor keeps retrying, the event processor will stall entirely when the cause is consistent.
+//
+//Although this behavior is sufficient on many occasions, sometimes it is beneficial if we can unblock event handling by parking the problematic event.
+//This is where the Dead-letter Queue comes in. It is a mechanism to park events until they can be processed successfully.
+//You can find more information in the xref:axon-framework-reference:events:event-processors/dead-letter-queue.adoc[] section.
 
 [[general_processor_configuration]]
 == General processor configuration
```

Comment:
The subsections here need heavy rewriting as well. Some parts in there still exist, while others have been removed.